pkgs/tools/audio/mp3cat/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "mp3cat-${version}";
+  version = "0.5";
+
+  src = fetchFromGitHub {
+    owner = "tomclegg";
+    repo = "mp3cat";
+    rev = version;
+    sha256 = "0n6hjg2wgd06m561zc3ib5w2m3pwpf74njv2b2w4sqqh5md2ymfr";
+  };
+
+  makeFlags = [
+    "PREFIX=${placeholder ''out''}"
+    "CC:=$(CC)"
```

Comment:
I've checked and I don't see `$(CC)` being used in the `Makefile` (it just uses `cc`) so we can remove this.